File: JavaScript/remove_duplicates.js
function removeDuplicates(array) {
    array.sort();
    let result = [];
    for (let i = 0; i < array.length; i++) {
        if (array[i] !== array[i+1]) {
            result.push(array[i]);
        }
    }
    return result;
}

console.log(removeDuplicates([1, 2, 3, 5, 1, 5, 9, 1, 2, 8]));

File: Python/remove_duplicates.py
def remove_duplicates(string):
    new_string = ""
    characters = []
    for i in range(len(string)):
        if string[i] not in characters or string[i] == " ":
            characters.append(string[i])
            new_string += string[i]
    return new_string

print(remove_duplicates("Loops are cool!!"))
        